src/frontend/komponenter/ManuellJournalfør/KnyttTilNyBehandling.tsx

```diff
@@ -38,7 +39,7 @@ export const KnyttTilNyBehandling: React.FC = () => {
                 />
             </StyledCheckboxDiv>
             {skjema.felter.behandlingstype.erSynlig && (
-                <>
+                <SkjemaGruppe>
```

Comment:
kjempefint at du tar tak i disse småfiksene! 

Et innspill du kan velge om du vil gjøre noe med (skjemagruppe i seg selv vil uansett være en semantisk oppgradering): 
Kanskje man kan trekke ut hele skjemagruppen til et OpprettBehandlingSkjema i felleskomponenter, hvor man inkluderer skjultlegend og feil, slik som i OpprettBehandling.tsx. Da kunne begge stedene dratt fordel av dette :) skjult legend for UU og feilmelding ved innsending.

Reply:
Jeg er usikker på om man får så godt feilmelding dersom opprettelse feiler ved journalføring nå . Du kan evt slå av backenden før du journalfører og se hva som skjer.

Reply:
Stemmer, dette her er feilmeldingen som kommer når jeg tester uten backend kjørende:
![image](https://user-images.githubusercontent.com/25459913/131836154-4e1cd440-7dd6-475b-b40c-bc73e0c034ec.png)